sound/soc/codecs/jz4770.c

```diff
@@ -368,17 +368,22 @@ static int hpout_event(struct snd_soc_dapm_widget *w,
 
 	switch (event) {
 	case SND_SOC_DAPM_PRE_PMU:
-		/* set cap-less, unmute HP */
+		/* wait for the DAC to be ready after DAC_SB=0 */
+		msleep(400);
+
+		/* unmute HP */
 		regmap_clear_bits(jz_codec->regmap, JZ4770_CODEC_REG_CR_HP,
-				  REG_CR_HP_SB_HPCM | REG_CR_HP_MUTE);
+                                  REG_CR_HP_MUTE);
```

Comment:
Keep the same alignment - align with tabs, and complete with spaces so that it's aligned with the jz_codec->regmap